readme/Contrib.scalatex

```diff
@@ -0,0 +1,69 @@
+@import Main._
+@import scalaworld.Readme._
+
+@sect{Part 6 - Contrib}
+  Scala.meta contrib is a module that provides common utilities for handling
+  scala.meta data structures.
+
+  @p
+    To use contrib, @code{import scala.meta.contrib._}.
+
+  @p
+    Contrib exposes some collection-like methods on @code{Tree}.
+    @meta
+      source"""
+      class A
+      trait B
+      object C
+      object D
+      """.find(_.is[Defn.Object])
+
+    @meta
+      source"""
+      class A
+      trait B
+      object C {
+        val x = 2
+        val y = 3
+      }
+      object D
+      """.collectFirst { case q"val y = $body" =>  body.structure }
+
+    @meta
+      source"""
+      class A
+      trait B
+      object C {
+        val x = 2
+        val y = 3
+      }
+      object D
+      """.exists(_.is[Defn.Def])
+
+  @p
+    Contrib has a @code{Equal} typeclass for comparing trees by structural or
+    syntactic equality.
```

Comment:
This will all need to be changed very soon. I suggest waiting for the equality merge and using the updated api.

Something like you can compare trees!

`q"val x = 2".isEqual(q"def x = 2")`

If you really require it. you case also compare syntax

... Some syntax test